write-fonts/Cargo.toml

```diff
@@ -29,7 +29,7 @@ font-test-data = { path = "../font-test-data" }
 read-fonts = { version = "0.17.0", path = "../read-fonts", features = [
   "codegen_test",
 ] }
-env_logger = "0.10.0"
```

Comment:
This seems like it should move to the workspace toml?

Reply:
👍 good idea